src/engine/FileManager.ts

```diff
@@ -332,8 +324,23 @@ class FileManager {
         }
     }
 
+    private getRecursiveListOfClsFiles(sourceDir: string, arrayOfFiles: string[] = [] as string[]): string[] {
+        const files = fs.readdirSync(sourceDir);
+        arrayOfFiles = arrayOfFiles || [];
```

Comment:
I don't think this line is needed since `arrayOfFiles` already defaults to an empty array